ssc/CMakeLists.txt

```diff
@@ -30,7 +30,6 @@ set(SSC_SRC
 		cmod_csp_common_eqns.cpp
 		cmod_csp_common_eqns.h
 		cmod_csp_dsg_lf_ui.cpp
-        cmod_csp_heatsink.cpp
```

Comment:
Why is this removed?

Reply:
Whoops - looks like I used --ours a little too aggressively on the merge. This is a good reason to do two PRs instead of combining this with changes to develop. I'll close this one.